services/backend/src/core/connections/connections.js

```diff
@@ -81,11 +74,9 @@ async function removeConnection(request, response) {
 async function getConnectionById(request, response) {
   try {
     validationResult(request).throw();
-
-    const { id } = request.params;
-    const userId = getKeycloakUserId(request);
-
-    if (userId && id) {
```

Comment:
It think this was removed due to merge conflict. this was changed 7 days ago by ben so i dont think it shoul dbe changed back